src/parser/core/modules/ActionWindow/windows/ActionWindow.tsx

```diff
@@ -20,7 +20,7 @@ import {History, HistoryEntry} from '../History'
 export abstract class ActionWindow extends Analyser {
 
 	@dependency protected data!: Data
-	@dependency private suggestions!: Suggestions
+	@dependency protected suggestions!: Suggestions
```

Comment:
This should return to private after making PlayersBuffedEvaluator do it's own suggestions.

```suggestion
	@dependency private suggestions!: Suggestions
```